about.md

```diff
@@ -20,6 +20,12 @@ Servo is written in [**Rust**](https://www.rust-lang.org/), taking advantage of
 
 Since its creation in 2012, Servo has contributed to W3C and WHATWG web standards, reporting specification issues and submitting new cross-browser automated tests, and core team members have co-edited new standards that have been adopted by other browsers. As a result, the Servo project helps drive the entire web platform forward, while building on a platform of reusable, modular technologies that implement web standards.
 
+## WPT pass rates
+
+The chart below tracks our pass rates in several *focus areas* of the [Web Platform Tests](http://web-platform-tests.org/), as well as the whole CSS and WPT test suites. To drill down the pass rates under a focus area, see the [Servo runs on wpt.fyi](https://wpt.fyi/runs?label=master&product=servo), or for more details and [legacy layout](/blog/2023/04/13/layout-2013-vs-2020/) scores, see our full WPT dashboard at [wpt.servo.org](https://wpt.servo.org).
```

Comment:
I thinking linking to results is easier to understand:

```suggestion
The chart below tracks our pass rates in several *focus areas* of the [Web Platform Tests](http://web-platform-tests.org/), as well as the whole CSS and WPT test suites. To drill down the pass rates under a focus area, see the [Servo results on wpt.fyi](https://wpt.fyi/results/?label=master&label=experimental&product=servo), or for more details and [legacy layout](/blog/2023/04/13/layout-2013-vs-2020/) scores, see our full WPT dashboard at [wpt.servo.org](https://wpt.servo.org).
```

Reply:
done in 09f1a75585e40d74cb0d1bb6adcd285dc072da70